k8s/resources/common/loadflow-server-service.yaml

```diff
@@ -3,6 +3,7 @@ kind: Service
 metadata:
   labels:
     name: loadflow-server
+    app-prometheus-metrics: true
```

Comment:
put label on all deployments ? (with common labels for example )

Reply:
In the first step we want to instrument only loadflow-server and then grow step-by-step that's why I didn't want to put the label on all of them straight away